src/dj_notebook/__init__.py

```diff
@@ -46,3 +60,15 @@ def activate(settings: str, quiet_load: bool = True) -> Plus:
             plus.print()
 
     return plus
+
+
+def is_notebook():
```

Comment:
I did not expect this to ever be called outside of a Jupyter notebook. Is that a thing for your team?